.github/workflows/master_update.yml

```diff
@@ -0,0 +1,18 @@
+name: master_update
+
+on:
+  push:
+    # branches:
+      # - master
+
+jobs:
+  release_image:
+    name: Build and publish to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: elgohr/Publish-Docker-Github-Action@master
```

Comment:
I never used a GitHub action for this, personally I prefer to make my own job pushing to CI, but this one doesn't look bad. I'd also take advantage of [caching](https://github.com/marketplace/actions/publish-docker#cache), to avoid unnecessary slowdowns.

So... no need to change anything here, LGTM, I just wanted to let you know about this 🙂